Tella/Components/BorderedTextEditorView.swift

```diff
@@ -86,12 +86,14 @@ struct BorderedTextEditorView: View {
     }
 }
 
-#Preview {
```

Comment:
I think the preview code `#Preview {` is automatically generated with xcode 15 and I think it's better to use the new syntax

Reply:
yeah, T changed that because I was using an older version of xcode and it didn't let me make a build. I will update the ones regarding feedback method since those are the new ones